docs_sources/erp_fhirversion_change_20250115-source.adoc

```diff
@@ -0,0 +1,117 @@
+= E-Rezept FHIR Veränderungen für 20250115 image:gematik_logo.png[width=150, float="right"]
+include::./config-source.adoc[]
+
+Ab 15.01.2025 ist das gematik Workflow Package 1.4.0 gültig. Da sowohl DiGA als auch eML Teil des Packages sind, müssen einige Veränderungen vorgenommen werden. Diese Seite bietet eine Zusammenfassung der Veränderungen für die verschiedenen Hersteller.
+
+toc::[]
+
+== Versionsübergabegänge
+Ab dem 01.11.2024 kann der E-Rezept-Fachdienst mehrere aktive Profile haben. Das bedeutet, dass Übergangszeiten mit mehr als zwei aktiven Profilen möglich sind. Unten sind sowohl ein Diagramm als auch eine Tabelle zu sehen, die zeigen, wann welches Profil aktiv ist und bis wann.
+
+image:puml_workflow_profile_version_timeline.png[]
```

Comment:
In der grünen bubble steht "Profile 1.3.1", sollte aber "Profile 1.3" sein oder?
In der Note würde ich dann noch "Package 1.3.1" schreiben, ist dann etwas eindeutiger.